.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,24 @@
+##### Issue type
+<!--- Pick one below and delete others -->
+ - Bug report
+ - Feature request
+ - Documentation issue
+ - Build issue
+ 
+##### Minetest version
+<!--- 
+Paste minetest version between quotes below
+If you are on a devel version, please add git commit hash
+-->
+```
+
+```
+
+##### OS / Hardware
+<!--- Give us informations about your system, operating system -->
```

Comment:
> Give us informations about your system, operating system

This could be better worded as:

> Give us information about your hardware and operating system